modules/monitoring/application/controllers/EventController.php

```diff
@@ -70,6 +72,14 @@ public function showAction()
             $this->getDetails($type, $event)
         );
 
+        $hookHtml = '';
+        /** @var EventDetailsExtensionHook $hook */
+        foreach (Hook::all('Monitoring\\EventDetailsExtension') as $hook) {
+            $hookHtml .= $hook->getHtmlForEvent($event);
+        }
+
+        $this->view->hookHtml = $hookHtml;
```

Comment:
Please do this the same way as it's done here:
https://github.com/Icinga/icingaweb2/blob/201cfa2330b826f862aa0dff23ebd552fa2fc38e/modules/monitoring/library/Monitoring/Web/Controller/MonitoredObjectController.php#L85-L102

i.e. Error handling, use a module container (otherwise a module's css is not applied)